src/Core/src/Graphics/Shape.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Graphics;
+
+namespace Microsoft.Maui.Graphics
+{
+	public abstract class Shape : IShape
```

Comment:
I think we don't need `Shape` as a class.   IShapeView will implement IShape and have these other properties, and a ShapeView in Controls can implement these as bindable.

Reply:
Removed.